### PR TITLE
#182 | Removed double newline in Newcomers_Guide.rst

### PR DESCRIPTION
**Pull request by bradarv90**
Fixes https://github.com/coala/documentation/issues/182

### CODE REVIEW

**Comment by meetmangukiya** [comment]
Amend the commit message 
> Removed double e newline in Newcomers_Guide.rst

to maybe **Newcomers_Guide: Remove double new line**

Also read https://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html

Thanks for the PR :smile:

**Comment by Adrianzatreanu** [comment]
also it appears you are not assigned to that issue, someone else is :/ could you try picking a different one maybe? thanks!

to prevent this in the future: let us know what issues you are working on so this doesnt happen again

**Comment by bradarv90** [comment]
@Adrianzatreanu That kinda seems silly. The person assigned was someone you assigned after I made the PR and that individual was told that they shouldn't work on newcomers tasks since they had already contributed before.

**Comment by sils** [comment]
Hey @bradarv90 sorry that this happened. We try to prevent this with assignments and those are the reasons we're very keen on those:

- Our newcomer issues are *made* for you people, they're so easy that they actually usually take more work to file than to fix them quickly for most of our experienced developers. We do this because we think it helps you and it costs a hell lot of time.
- Currently we have over 30 newcomer issues assigned - and sometimes there's a lot of people asking for assignment at the same time. Because of the sheer amount of work with all the newcomers we're unable to do much more than following our process - so if someone asks for an assignment on an unassigned issue he gets it and sometimes people don't look for open PRs for an issue.

In short: too much work, undermanned and we're all volunteers. There's no commercial backing.

I'm sorry that this happened and yes it would have been avoidable if there was more time but we don't have the manpower to do that.
